storage/google_drive/drive_client_wrapper.py

```diff
@@ -0,0 +1,135 @@
+from googleapiclient.discovery import build
+from httplib2 import Http
+from oauth2client import file, client, tools
+from googleapiclient.http import MediaFileUpload
+
+import mimetypes
+import os
+
+# If modifying these scopes, delete the file token.json.
+SCOPES = 'https://www.googleapis.com/auth/drive'
+
+drive_service = None
+drive_folder_root = None
```

Comment:
Swapped it round - does this make a bit more sense?

It's basically a shortcut so that I don't have to call drive_service.files() everytime, though looking at it now, it's not that much of a shortcut... Happy to remove it if you think it would be better